src/CodeGenerator/InjectorGenerator.php

```diff
@@ -57,48 +55,40 @@ class InjectorGenerator
      */
     protected $definition;
 
-    /**
-     * @var string
-     */
+    /** @var string */
     private $namespace;
 
-    /**
-     * @var FactoryGenerator
-     */
+    /** @var FactoryGenerator */
     private $factoryGenerator;
 
-    /**
-     * @var AutoloadGenerator
-     */
+    /** @var AutoloadGenerator */
     private $autoloadGenerator;
 
-    /**
-     * @var LoggerInterface
-     */
+    /** @var LoggerInterface */
     private $logger;
 
     /**
      * Constructs the compiler instance
      *
-     * @param ConfigInterface $config The configuration to compile from
-     * @param DependencyResolverInterface $resolver The resolver to utilize
-     * @param string|null $namespace Namespace to use for generated class; defaults
-     *     to Zend\Di\Generated.
-     * @param LoggerInterface|null $logger An optional logger instance to log failures
-     *     and processed classes.
+     * @param ConfigInterface             $config    The configuration to compile from
+     * @param DependencyResolverInterface $resolver  The resolver to utilize
+     * @param string|null                 $namespace Namespace to use for generated class; defaults
+     *                     to Zend\Di\Generated.
+     * @param LoggerInterface|null        $logger    An optional logger instance to log failures
+     *               and processed classes.
```

Comment:
Hm... these indent of second line of the comment are weird. Is it automatically fixed?

@xtreamwayz I think we need to check it.